public/static/docs/tutorials/pipelines.md

```diff
@@ -51,7 +51,7 @@ $ git commit -m "Download and add code to new Git repo"
 > `dvc get` can use any <abbr>DVC project</abbr> hosted on a Git repository to
 > find the appropriate [remote storage](/doc/command-reference/remote) and
 > download <abbr>data artifacts</abbr> from it. (It works like `wget`, but for
-> DVC repositories.) In this case we use
+> DVC or Git repositories.) In this case we use
```

Comment:
I'll change this back to DVC repo and put `<abbr>` around it so the tooltip (which mentions DVC repos are Git repos) comes up.

Reply:
See 96232e0a